doc/changelog.rst

```diff
@@ -7,7 +7,15 @@ v2.13.0
 ------------------------
 Highlights
```

Comment:
The highlights section doesn't look "highlighted" :smile: You can underline the title section with `~~~~` as in other sections